ci/authn-oidc/phantomjs/keycloak_login.js

```diff
@@ -8,38 +8,42 @@ var webPage = require('webpage');
 var page = webPage.create();
 var system = require('system');
 var env = system.env;
+var fs = require('fs');
+var date = Date.now().toString();
+var logfile = 'keycloak_login.' + date + '.log';
+log('Date in container: ' + date);
 page.settings.userAgent = 'Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/44.0.2403.157 Safari/537.36';
 page.settings.javascriptEnabled = true;
 page.settings.loadImages = false;//Script is much faster with this field set to false
 phantom.cookiesEnabled = true;
 phantom.javascriptEnabled = true;
 /*********SETTINGS END*****************/
 
-console.log('All settings loaded, start with execution');
+log('All settings loaded, start with execution');
```

Comment:
maybe write what settings were loaded?

Reply:
It is not really interesting, this is only an help script to fetch the authorization code